packages/nuxt/src/module.ts

```diff
@@ -28,13 +29,15 @@ export default defineNuxtModule<SdkModuleOptions>({
       enabled: false,
     },
   },
-  setup(options, nuxt) {
+  async setup(options, nuxt) {
     const projectLayer = nuxt.options._layers[0];
 
     const projectRootResolver = createResolver(projectLayer.config.rootDir);
     const buildDirectoryResolver = createResolver(nuxt.options.buildDir);
     const localResolver = createResolver(import.meta.url);
 
+    await installModule("@pinia/nuxt");
+
     nuxt.options.runtimeConfig.public.alokai = defu(
       nuxt.options.runtimeConfig.public?.vsf as any,
```

Comment:
If we're making breaking change release, please remove this line - we shoudn't support the vsf prefix in our environment variables anymore

Reply:
As well as line 49, 50

Reply:
Changed